### DISCUSSION TITLE
`[调试模式]工程列表搜索最终文本：`

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#3415] [讨论]
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/fe27b14c-7b59-4e54-a4c9-ab3596359093)
用MOD、整合包搜索功能的时候突然弹了个这调试信息吓了我一跳（因为以前好像没有吧...）
~怎么说呢还挺智能~

**Comment by allMagicNB** [comment]
你这应该是讨论吧……

**Reply by kong-que-dnf123** [reply]
额那我改一下...

**Comment by MoYuan-CN** [comment]
这是你开了调试模式（PCL -> 设置 -> 启动器 -> 高级 -> 调试模式）
开了之后他会返回你搜索的内容经过优化之后实际发送给 CF/MR 进行搜索的文本
